src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball [] balls = new Ball[3];
```

Comment:
Let's not create a separate array for Ball instances, when you can just call getRandomBall() and immediately print it out.